src/applications/vre/28-8832/containers/IntroductionPage.jsx

```diff
@@ -8,153 +8,172 @@ import { focusElement } from 'platform/utilities/ui';
+        Equal to VA Form 28-8832 (Education/Vocational Counseling Application).
+      </p>
+      <SaveInProgressIntro
+        prefillEnabled={props.route.formConfig.prefillEnabled}
+        messages={props.route.formConfig.savedFormMessages}
+        pageList={pageList}
+        downtime={props.route.formConfig.downtime}
+        startText="Apply for career planning and guidance"
+      >
+        Please complete the 28-8832 form to apply for Planning and career
+        guidance.
+      </SaveInProgressIntro>
+      <h4>Follow the steps below to apply for career planning and guidance.</h4>
+      <div className="process schemaform-process">
+        <ol>
+          <li className="process-step list-one">
+            <h5>Prepare</h5>
+            <h6>To fill out this application, you’ll need your:</h6>
+            <ul>
+              <li>Social Security number</li>
+              <li>Date of birth</li>
+              <li>
+                If you're a dependent, you'll need the name and Social Security
+                number of the Veteran or service member who sponsors you.
+              </li>
+            </ul>
+            <p>
+              <strong>What if I need help filling out my application?</strong>{' '}
+              An accredited representative, with a Veterans Service Organization
+              (VSO), can help you fill out your claim.{' '}
+              <a href="/disability-benefits/apply/help/index.html">
+                Get help filing your claim
+              </a>
+            </p>
+          </li>
+          <li className="process-step list-two">
+            <h5>Apply</h5>
+            <p>Complete this career planning and guidance form.</p>
+            <p>
+              After submitting your application, you'll get a confirmation
+              message. It will include details about your next steps. You can
+              print this for your records.
+            </p>
+          </li>
+          <li className="process-step list-three">
+            <h5>VA Review</h5>
+            <p>
+              We process applications in the order we receive them. We may
+              contact you if we have questions or need more information.
+            </p>
+          </li>
+          <li className="process-step list-four">
+            <h5>Decision</h5>
+            <p>
+              If you’re eligible for career planning and guidance benefits,
+              we’ll invite you to an orientation session at your nearest VA
+              regional office.
+            </p>
+          </li>
+        </ol>
         <p>
-          Equal to VA Form 28-8832 (Education/Vocational Counseling
-          Application).
+          If you're not sure this is the right form, you can{' '}
+          <a
+            href={`${PCPG_ROOT_URL}/introduction`}
+            onClick={() => {
+              recordEvent({
+                event: 'howToWizard-start-over',
+              });
+              sessionStorage.removeItem(WIZARD_STATUS);
```

Comment:
This should cause a new render so I think the `userEffect` should take care of moving the focus, but it would be good to verify.